examples/swift-ios/bugsnag-example/BigHonkinWebViewController.h

```diff
@@ -0,0 +1,4 @@
+#import <UIKit/UIKit.h>
+@interface BigHonkinWebViewController : UIViewController
```

Comment:
I'm fairly sure the the big honker was removed in the past because we weren't keen on the frivolous naming.  Could you rename the files to indicate their actual purpose?

Reply:
Name changed to `OutOfMemoryController.m/h`, references all updated.